### PR TITLE
Add secretary form

### PR DESCRIPTION
**Pull request by Vinnytile**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.less [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  max-width: 310px;
+}
+.ant-picker {
+  width: 310px;
+}
+
+.formItemExplain {
+  width: 310px;
+  float: right;
+}
```
Comment (Oleh-Bodnar) [inline src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.less:11]: Add an empty row at the end of the file please)

File: src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.less [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  max-width: 310px;
+}
+.ant-picker {
+  width: 310px;
+}
+
+.formItemExplain {
+  width: 310px;
+  float: right;
+}
```
Comment (Oleh-Bodnar) [inline src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.less:11]: Add an empty row at the end of the file please)

File: src/pages/Regions/AddRegionSecretaryForm.less [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  max-width: 310px;
+}
+.ant-picker {
+  width: 310px;
+}
+
+.formItemExplain {
+  width: 310px;
+  float: right;
+}
```
Comment (Oleh-Bodnar) [inline src/pages/Regions/AddRegionSecretaryForm.less:11]: Add an empty row at the end of the file please)

File: src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.less (hunk 1 above)
Comment (Vinnytile) [reply src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.less]: Done

File: src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.less (hunk 2 above)
Comment (Vinnytile) [reply src/pages/Club/AddAdministratorModal/AddClubsSecretaryForm.less]: Done

File: src/pages/Regions/AddRegionSecretaryForm.less (hunk 3 above)
Comment (Vinnytile) [reply src/pages/Regions/AddRegionSecretaryForm.less]: Done

**Review by Oleh-Bodnar — APPROVED** [review]

**Review by tanyabiryuk — APPROVED** [review]

**Review by veronika-kolosova — APPROVED** [review]

**Pull request merged**